pom.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.unicode.unicodetools</groupId>
+	<artifactId>unicodetools-parent</artifactId>
+	<version>0.0.0</version>
```

Comment:
Can we have a non-zero version number? (It seems strange to have v.0.0.0) Even if we don't start at 1.0.0, we should at least start at 0.1.0.  (And of course, the subproject pom.xml versions must be kept in lockstep with their parent pom.xml.)

Reply:
1.0.0 it is!

Reply:
looks fixed (1.0.0) in the last commit